src/DatePicker/DatePicker.less

```diff
@@ -0,0 +1,87 @@
+@import (reference) "../less/index";
+@import "~react-datepicker/dist/react-datepicker.min.css";
+
+.DatePicker {
+  background-color: @neutral_off_white;
+  position: absolute;
+  margin-top: @size_2xs;
+  .border--s(@neutral_silver);
+  z-index: @zIndex2;
+}
+
+.DatePicker--header {
+  padding: @size_xs;
+  .border--bottom--s(@neutral_silver);
+  color: @neutral_dark_gray;
+}
+
+.DatePicker--title {
+  font-size: @type_small;
```

Comment:
Use mixins: .text--small, .text--semi-bold.

We should rarely use actual bold.